pkg/nsx/services/common/init.go

```diff
@@ -0,0 +1,32 @@
+package common
+
+import (
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/bindings"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/vmware-tanzu/nsx-operator/pkg/config"
+	"github.com/vmware-tanzu/nsx-operator/pkg/nsx"
+)
+
+type Service struct {
+	NSXClient *nsx.Client
+	NSXConfig *config.NSXOperatorConfig
+}
+
+const (
+	PageSize                   int64 = 1000
```

Comment:
I see PageSize was defined as a Var instead of const. Is it ok to change it to const?

Reply:
In store.go
	pageSize := common.PageSize
so it is ok.